### PR TITLE
update links (http=>https)

### PR DESCRIPTION
**Pull request by Arhell**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: docs/guide-ja/basic-usage.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ Gii にアクセスすると、最初に、ジェネレータを選択できる
 その場合は、どのファイルを上書きし、どのファイルを上書きしないかを選択することも出来ます。
 
 > Tip: データベースに変更を加えた後で、モデル・ジェネレータを使ってモデルを更新したい場合は、Gii のプレビューからコードをコピーして、あなたのコードに変更をマージすることが出来ます。
-PHPStorm の [クリップボードと比較](http://www.jetbrains.com/phpstorm/webhelp/comparing-files.html) のような IDE の機能を使うと良いでしょう。
-[Aptana Studio](http://www.aptana.com/products/studio3/download) や [Eclipse](http://www.eclipse.org/pdt/) ベースのエディタも [AnyEdit tools plugin](http://andrei.gmxhome.de/anyedit/) を使って [クリップボードと比較](http://andrei.gmxhome.de/anyedit/examples.html) をすることが出来ます。
+PHPStorm の [クリップボードと比較](https://www.jetbrains.com/phpstorm/webhelp/comparing-files.html) のような IDE の機能を使うと良いでしょう。
```
Comment (vjik) [inline docs/guide-ja/basic-usage.md:28]:
```suggestion
PHPStorm の [クリップボードと比較](https://www.jetbrains.com/help/phpstorm/comparing-files-and-folders.html) のような IDE の機能を使うと良いでしょう。
```

File: docs/guide-ja/basic-usage.md [hunk 2]
```diff
@@ -25,8 +25,8 @@ Gii にアクセスすると、最初に、ジェネレータを選択できる
...
+[Aptana Studio](https://www.aptana.com/products/studio3/download) や [Eclipse](https://www.eclipse.org/pdt/) ベースのエディタも [AnyEdit tools plugin](https://andrei.gmxhome.de/anyedit/) を使って [クリップボードと比較](https://andrei.gmxhome.de/anyedit/examples.html) をすることが出来ます。
```
Comment (vjik) [inline docs/guide-ja/basic-usage.md:29]:
```suggestion
[Aptana Studio](http://www.aptana.com/) や [Eclipse](https://www.eclipse.org/pdt/) ベースのエディタも [AnyEdit tools plugin](https://andrei.gmxhome.de/anyedit/) を使って [クリップボードと比較](https://andrei.gmxhome.de/anyedit/examples.html) をすることが出来ます。
```

Comment (vjik) [reply docs/guide-ja/basic-usage.md:29]: https://andrei.gmxhome.de/ — broken link

**Review by vjik — CHANGES_REQUESTED** [review]

File: docs/guide-ja/basic-usage.md (hunk 1 above)
Comment (Arhell) [reply docs/guide-ja/basic-usage.md]: Done thanks

File: docs/guide-ja/basic-usage.md (hunk 2 above)
Comment (Arhell) [reply docs/guide-ja/basic-usage.md]: Done thanks

**Review by vjik — APPROVED** [review]

**Pull request merged**